### ISSUE TITLE
Contrast issues

### ISSUE THREAD

**Issue by jarmitage** [disastrid/instruments_lab#13]
According to Web Content Accessibility Guidelines (WCAG) 2 we should have a minimum contrast ratio of 4.5:1 for text

https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#contrast-minimum
https://webaim.org/resources/contrastchecker/

![screenshot 2018-12-12 at 14 49 16](https://user-images.githubusercontent.com/2885827/49882175-f442d380-fe27-11e8-9629-45e321030b1c.png)
